php/README.md

```diff
@@ -98,24 +96,18 @@ Known Issues
 
 ## Development
 
-### Docker Image
```

Comment:
Because this statement seemed outdated.  It's no longer used in our automated tests and rather than update it to work with bazel I figured it was best to just delete it (we had a conversation about this in the on-duty chat a while back).

If you think this is still valuable I'm happy to add it back though.  I still need to do a second pass of all the markdown files